src/modules/business/services/eventBusiness/DeleteEventBusinessService.ts

```diff
@@ -0,0 +1,29 @@
+import { injectable, inject } from 'tsyringe';
+
+import AppError from '@shared/infra/http/errors/AppError';
+
+import IEventBusinessRepository from '../../repositories/IEventBusinessRepository';
+
+interface IRequest {
+  event_business_id: string;
+}
+
+@injectable()
+export default class DeleteEventBusinessService {
+  constructor(
+    @inject('EventBusinessRepository')
+    private eventBusinessRepository: IEventBusinessRepository,
+  ) {}
+
+  public async execute({ event_business_id }: IRequest): Promise<void> {
+    const eventBusiness = await this.eventBusinessRepository.findById(
+      event_business_id,
+    );
+
```

Comment:
just delete if the business_id is the same as the business_id registered at the event